src/pages/settings/SettingsPage.tsx

```
@@ -0,0 +1,94 @@
+  const navigate = useNavigate();
+  const [isProfilePickerOpen, setIsProfilePickerOpen] = useState(false);
+  const [isLogoutOpen, setIsLogoutOpen] = useState(false);
+  const [isWithdrawOpen, setIsWithdrawOpen] = useState(false);
+
+  // TODO(API 연결 시): user / profiles를 query로 교체
+  const user = mockUser;
+  const profiles = mockProfiles;
+  const [selectedProfileId, setSelectedProfileId] = useState(user.profileId);
+
+  const selectedProfileLabel = useMemo(() => {
+    return profiles.find((p) => p.id === selectedProfileId)?.label ?? "프로필";
```

Biome (2.3.13)

[error] This hook specifies more dependencies than necessary: profiles.

This dependency can be removed from the list.

React relies on hook dependencies to determine when to re-compute Effects.
Specifying more dependencies than required can lead to unnecessary re-rendering
and degraded performance.
Unsafe fix: Remove the extra dependencies from the list.


(lint/correctness/useExhaustiveDependencies)